CHANGES.md

```diff
@@ -24,6 +24,19 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.5.0
+
+- new features
```

Comment:
shouldn't we break this between raiwidgets and responsibleai packages as separate release notes?  Also, this isn't how we released responsibleai package last time, it's not part of the release pipeline currently.